input/blog/2020-11-05-cake-v1.0.0-rc0001-released.md

```diff
@@ -0,0 +1,234 @@
+﻿---
+title: Cake v1.0.0 RC1 released
+category: Release Notes
+author: devlead
+---
+
+Version 1.0.0-rc0001 of Cake has been released
+
+This version is the 110th release of Cake, but the first pre-release of Cake 1.0🎉
+In this blog post, we will go through some of the major new features of 1.0, which platforms are supported, and how **you** can help to get Cake 1.0 released.
+
+### Why Release Candidate?
+
+We've chosen to release 1.0 as a pre-release to NuGet and Chocolatey, so we and the community easier can test and ensure that our 1.0 release will be as stable as possible.
+It will also enable us to see if there are any steps you need to take to adapt your scripts for 1.0, and for us to see if there're any unintended breaking changes, test as many addins, modules, and recipes as possible.
```

Comment:
```suggestion
It will also enable us to see if there are any steps you need to take to adapt your scripts for 1.0, and for us to see if there're any unintended breaking changes, by testing as many addins, modules, and recipes as possible.
```